NetworkIOMeter.c

```diff
@@ -80,6 +80,12 @@ static void NetworkIOMeter_updateValues(ATTR_UNUSED Meter* this, char* buffer, s
       cached_txp_total = packetsTransmitted;
    }
 
+   this->values[0] = cached_rxb_diff;
+   this->values[1] = cached_txb_diff;
+   if (cached_rxb_diff + cached_txb_diff > this->total) {
+      this->total = cached_rxb_diff;
```

Comment:
Why is the total not `cached_rxb_diff + cached_txb_diff`?

Reply:
fixed it.